### ISSUE TITLE
move metapackages to conda-forge

### ISSUE THREAD

**Issue by scottyhq** [pangeo-data/conda-metapackages#1]
metapackages are currently hosted in the pangeo/ anaconda cloud channel. It would be nice to move to conda-forge to avoid needing to specific the extra channel for conda operations:

```
name: pangeo
channels:
  - conda-forge
  - pangeo
dependencies:
  - python=3.7
  - pangeo-notebook=0.0.2
```

@jhamman and @ocefpaf could provide guidance on this

**Comment by ocefpaf** [comment]
See https://github.com/conda-forge/staged-recipes/pull/10969

**Comment by scottyhq** [comment]
closed by conda-forge/staged-recipes#10969
